.github/workflows/scripts/addClosingMessage.js

```diff
@@ -61,15 +61,17 @@ module.exports = async ({ core, context, github }) => {
       body,
     });
 
-    const labelName = 'status: waiting for maintainer';
-    core.debug(`>>> Removing label: ${labelName}`);
+    if (issue.data.labels.some((label) => label.name === 'status: waiting for maintainer')) {
+      const labelName = 'status: waiting for maintainer';
+      core.debug(`>>> Removing label: ${labelName}`);
 
-    await github.rest.issues.removeLabel({
-      owner,
-      repo,
-      issue_number: issueNumber,
-      name: labelName,
-    });
+      await github.rest.issues.removeLabel({
+        owner,
+        repo,
+        issue_number: issueNumber,
+        name: labelName,
+      });
+    }
```

Comment:
Probably better to `try/catch` it in (unlikely)case the label was added between checking and removing.

Reply:
mmh, could be ... lets see

Reply:
done

Reply:
ill not fail the job, but log the errormessage instead